src/js/jsx/PanelSet.jsx

```diff
@@ -279,10 +279,21 @@ define(function (require, exports, module) {
                 );
             } else if (this.state.recentFilesInitialized) {
                 return (
-                    <div ref="panelSet" className="panel-set__container">
-                        <div className="panel panel__visible">
+                    <div className="panel-set panel-set__active null-state">
+                        <PanelColumn
```

Comment:
The neighboring elements, when they have only one attribute, are kept to one line.. so maybe do that here